include/boost/geometry/algorithms/detail/sections/sectionalize.hpp

```diff
@@ -160,16 +160,14 @@ template
 >
 struct get_direction_loop
 {
-    typedef typename util::sequence_element<Index, DimensionVector>::type dimension;
+    using dimension = typename util::sequence_element<Index, DimensionVector>::type;
 
     template <typename Segment>
     static inline void apply(Segment const& seg,
                 int directions[Count])
```

Comment:
this is unrelated but you can also fix the indentation here.

Reply:
Yes, 1-tab it should have been I think.

`clang-format` is for 90% nice indeed. But the last 10% is awful, from all what I've seen, and it destroys readability which we carefully wrote everywhere (nearly).

I'm quite sensible to code layout and we try to do it carefully with optimum readability. Such as this one

https://github.com/boostorg/geometry/blob/develop/include/boost/geometry/srs/projections/impl/dms_parser.hpp#L249-L257

If you handle this with clang-format, it's destroyed. The same for streaming:

https://github.com/boostorg/geometry/blob/develop/include/boost/geometry/algorithms/simplify.hpp#L133-L151

This is also destroyed by clang-format.

But clang-format has many options, and I'm not an expert. If you have a configuration which works well in all cases, we can consider that.